src/toil/lib/conversions.py

```diff
@@ -1,4 +1,6 @@
+"""Conversions.py is now for general conversion functions as well as of July 13 2021"""
```

Comment:
Should be an extension of the original doc string, or comment.

Reply:
This has been updated in new commit